firefox-install-demo.toml

```diff
@@ -33,9 +40,46 @@ select_expression = "IF(LOGICAL_OR(succeeded), 1, 0) * 100"
 type = "scalar"
 
 [metrics.install_success_rate.statistics]
-sum = {}
 mean = {}
 
+[metrics.install_volume_by_os_version]
+data_source = "firefox_installs"
+select_expression = "IF(LOGICAL_OR(succeeded), 1, 0) * 100"
+type = "scalar"
+
+[metrics.install_volume_by_os_version.statistics]
+sum = {}
+
+[metrics.install_volume_total]
+data_source = "firefox_installs_all_versions"
+select_expression = "IF(LOGICAL_OR(succeeded), 1, 0) * 100"
+type = "scalar"
```

Comment:
Something I missed mentioning before, if the client population is segmented by their branches (which happens when `branches` gets defined under `population`) then _all_ graphs will be pivoted on the `branches`. So computing a total doesn't work at the moment.
It's definitely something that should be possible to do though, I opened a ticket to follow-up on this: https://github.com/mozilla/opmon/issues/74

It is possible to extend the generated Looker dashboard an add the graph manually there, however manually added graphs will also need manually defined alerts

Reply:
Thank you for the explanation!